app/views/shared/_footer.html.erb

```diff
@@ -0,0 +1,83 @@
+<div id="footer">
+  <div class="container-fluid">
+      <div class="row">
+        <div class="col-md-3">
+          <div class="content portlet-0">
+            <p><span>This History Project at UC Davis</span><br />
+              <span>One Shields Ave, 163 Kerr Hall, Davis, CA, 95616</span><br />
+              <span></span>tel: 530.752.4383<br />fax: 530.752.5655<br />
+              <a class="email-link" href="mailto:historyproject@ucdavis.edu" target="_self" title="">historyproject@ucdavis.edu</a>
+            </p>
+          </div>
+        </div>
+        <div class="col-md-3">
+          <div class="content portlet-1">
+            <div class="column">
+                <ul class="navTree navTreeLevel0">
+                  <li class="navTreeItem visualNoMarker section-programs">
+                    <a href="https://historyproject.ucdavis.edu/programs" class="state-published contenttype-link" title="">
```

Comment:
`state-published` and `contenttype-link` are not defined in CSS. Remove them if they're not necessary.